pkg/volume/csi/csi_mounter.go

```diff
@@ -207,9 +207,8 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 
 	if err != nil {
 		glog.Errorf(log("mounter.SetupAt failed: %v", err))
-		if err := removeMountDir(c.plugin, dir); err != nil {
-			glog.Error(log("mounter.SetuAt failed to remove mount dir after a NodePublish() error [%s]: %v", dir, err))
-			return err
+		if clearErr := removeMountDir(c.plugin, dir); clearErr != nil {
+			glog.Error(log("mounter.SetuAt failed to remove mount dir after a NodePublish() error [%s]: %v", dir, clearErr))
```

Comment:
"mounter.SetuAt" should be "mounter.SetupAt". Know its not your fault but might as well fix it since its here.

Reply:
Already fixed, PTAL.